DISCLAIMER.md

```diff
@@ -0,0 +1,49 @@
+# Disclaimer
+
+Simple Icons asks that its users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Licenses, Copyrights & Trademarks](#licenses-copyrights--trademarks)
+* [Brand Guidelines](#brand-guidelines)
+* [Update of Brands](#update-of-brands)
+* [Removal of Brands](#removal-of-brands)
+
+## Licenses, Copyrights & Trademarks
+
+> NOTE: the addition of licenses to Simple Icons is an ongoing project. Hence, the absence of licence data for a particular icon does not imply that the icon is not released under a license.
+
+Simple Icons provides data on the license under which icons are available. We ask users to carefully consider this when using an icon. Users of Simple icons are expected to request permission to use a brand's icon when necessary. As licenses are subject to change we also ask our users to regularly check if the license of the icons they use have been changed.
```

Comment:
I'd move the sentence about permissions either to the end of this section or into the guidelines section as it's usually in the guidelines that we'll find the explicit requirement to seek permission.